src/med_bench/mediation.py

```diff
@@ -554,7 +547,7 @@ def mediation_multiply_robust(y, t, m, x, interaction=False, forest=False,
     indirect1 = np.mean(y1m1 - y1m0)
     indirect0 = np.mean(y0m1 - y0m0)
 
-    return total, direct1, direct0, indirect1, indirect0, n_discarded
+    return [total, direct1, direct0, indirect1, indirect0, n_discarded]
```

Comment:
We discussed this with @judithabk6, apparently there was no specific reason to return a tuple instead of a list. It was creating an error to be handled in some tests, hence I thought it was cleaner to return a list in every estimator